Rebus.AmazonS3.Tests/AmazonS3ConnectionInfoUtil.cs

```diff
@@ -17,6 +17,7 @@ public class AmazonS3ConnectionInfoUtil
             {
                 try
                 {
+                    Console.WriteLine(file);
```

Comment:
but why?

Reply:
😄

Reply:
I could not understand why my unit tests on CI were failing and could not figure out how this connection info was resolved in CI environment. Turns out I wasn't providing the options with bucket name to my AmazonS3SubscriptionStorage class. I've removed this ConsoleWriteLine(file); already and pushed a fix to satisfy broken CI tests.

Reply:
Sorry about this check in. I promise not to use your aws credentials :)